group_vars/ppa/vars.yml

```diff
@@ -4,8 +4,8 @@
 ---
 # Using python35 and nodejs10 for QA and staging
 # Paths are picky and spaces for any YAML multiline syntax causes issues
-path: '/opt/rh/rh-python35/root/usr/bin:/opt/rh/rh-nodejs10/root/usr/bin:{{ ansible_env.PATH }}'
-ld_library_path: '/opt/rh/rh-python35/root/usr/lib64:/opt/rh/rh-nodejs10/root/usr/lib64{% if ansible_env.LD_LIBRARY_PATH is defined %}:{{ ansible_env.LD_LIBRARY_PATH }}{% endif %}'
+path: '/opt/rh/rh-python35/root/usr/bin:/opt/rh/rh-nodejs10/root/usr/bin:/opt/rh/rh-git227/root/bin:{{ ansible_env.PATH }}'
+ld_library_path: '/opt/rh/rh-python35/root/usr/lib64:/opt/rh/rh-nodejs10/root/usr/lib64:/opt/rh/httpd24/root/usr/lib64{% if ansible_env.LD_LIBRARY_PATH is defined %}:{{ ansible_env.LD_LIBRARY_PATH }}{% endif %}'
```

Comment:
seems like it would be nice to parameterize these more so that you could individually override python, node, or git versions for apps instead of having to redefine the whole string.

Reply:
💯  agree, but these will all go away when we get off RC/Springdal